src/test/scala/com/amazon/deequ/VerificationSuiteTest.scala

```diff
@@ -781,6 +781,61 @@ class VerificationSuiteTest extends WordSpec with Matchers with SparkContextSpec
           assert(checkConstraint == checkResultConstraint.constraint)
       }
     }
+
+    "A well-defined check should pass even if an ill-defined check is also configured" in withSparkSession {
+      sparkSession =>
+        val df = getDfWithNameAndAge(sparkSession)
+
+        val checkThatShouldSucceed =
+          Check(CheckLevel.Error, "shouldSucceedForValue").isComplete("name")
+
+        val complianceCheckThatShouldSucceed =
+          Check(CheckLevel.Error, "shouldSucceedForAge").isContainedIn("age", 1, 100)
+
+        val complianceCheckThatShouldFailForAge =
+          Check(CheckLevel.Error, "shouldFailForAge").isContainedIn("age", 1, 19)
+
+        val checkThatShouldFail = Check(CheckLevel.Error, "shouldErrorColumnNA")
+          .isContainedIn("fakeColumn", 10, 90)
+
+        val complianceCheckThatShouldFail = Check(CheckLevel.Error, "shouldErrorStringType")
+          .isContainedIn("name", 1, 3)
+
+        val complianceCheckThatShouldFailCompleteness = Check(CheckLevel.Error, "shouldErrorStringType")
+          .hasCompleteness("fake", x => x > 0)
```

Comment:
Can you also add a failing check for `isComplete` with a single column? I think that will return a completenessCheck (rather than a complianceCheck), and we should test that too.

Right now, completeness does have that precondition:
https://github.com/awslabs/deequ/blob/master/src/main/scala/com/amazon/deequ/analyzers/Completeness.scala#L47

But it would make sense for this test to fail if we remove it.